### ISSUE TITLE
[Bug]: The towhee.text_embedding model can not work on gpu? How to fix it?

### ISSUE THREAD

**Issue by check-777** [towhee-io/towhee#2324]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I do all operators about text_embedding.transformers(). , its always on CPU
--

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93):
- OS(Ubuntu or CentOS):
- CPU/Memory:
- GPU:
- Others:
```

**Comment by binbinlv** [comment]
@check-777 
could you please paste the error reported on your machine?

**Comment by jaelgu** [comment]
@check-777 

You can try this with latest op:
```
text_embedding.transformers(model_name='xxx', device='cuda:0')
```
If it says no attribute "device", then clean towhee cache by removing `rm -rf ~/.towhee/operators/text-embedding/transformers`

**Comment by check-777** [comment]
I fix this problem by using HuggingFace transformer model instead of text_embedding.transformers
`device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
model = AutoModel.from_pretrained('bert-base-chinese').to(device)
model.eval()
tokenizer = AutoTokenizer.from_pretrained('bert-base-chinese')
def Bert_Chinese(title):
    inputs =  tokenizer(title, return_tensors="pt").to(device)
    outs =  model(**inputs)
    features = outs['pooler_output'].squeeze(0)
    vec = features.cpu().detach().numpy()
    return vec`
